build.gradle

```diff
@@ -32,6 +33,14 @@ compileTestKotlin {
 }
 
 apply plugin: 'maven'
+apply plugin: 'info.solidsoft.pitest'
+
+pitest {
+    targetClasses = ['qbit.*']  //by default "${project.group}.*"
+    threads = 4
+    outputFormats = ['XML', 'HTML']
```

Comment:
Remove XML format, if it will not break anything

Reply:
Done